src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- Fixed a bug where `RESET (ttl_expire_after)` could incorrectly remove `ttl_expiration_expression`. [#110746][#110746]
+- Fixed a bug where an [`ALTER TABLE ... ADD CONSTRAINT CHECK ...`](https://www.cockroachlabs.com/docs/v23.1/alter-table#add-constraint) statement that utilized a user-defined function in the `CHECK` could cause a validation error. [#110720][#110720]
+- Fixed a bug where `CREATE INDEX` for [partial indexes](https://www.cockroachlabs.com/docs/v23.1/partial-indexes) could fail with `ERROR: duplicate key value violates unique constraint` if concurrent inserts happened simultaneously. [#110584][#110584]
+- Fixed a bug where a [`CREATE TABLE`](https://www.cockroachlabs.com/docs/v23.1/create-table) command with an `IDENTITY` column did not properly propagate the type of the column into the sequence. [#111014][#111014]
+- Fixed a bug where the `format_type` built-in did not honor `typemod` information for array types, leading to incorrect output. [#110940][#110940]
+- Fixed compaction behavior to prevent rapid sublevel growth when removing replicas from a store, such as during decommissioning. [#111141][#111141]
+- Fixed a bug that could occur when the "multiple active portals" execution mode (Preview) was enabled to evaluate queries such as lookup joins. The bug could result in an internal error like `unexpected 40960 leftover bytes` if the portal was not fully consumed. [#110666][#110666]
+- External connection URLs now accept the scheme `azure-blob` for connections to Azure Blob Storage and the scheme `azure-kms` for connections to Azure KMS. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage. [#111246][#111246]
+- Fixed a bug where changing the setting `server.telemetry.hot_ranges_stats.interval` had no effect. [#111373][#111373]
+- Added a check for values before using `mean` on the [Plan Details page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page), fixing a crash. [#111505][#111505]
+- Fixed a bug where dependencies on sequences from tables would be reported with the wrong value for the `classid` column in the `pg_catalog.pg_depend` table. [#111600][#111600]
+- Fixed a bug that could cause a transaction performing multiple parallel foreign key checks to return a `concurrent txn use detected` error. [#111600][#111600]
+- Removed buggy [TTL](https://www.cockroachlabs.com/docs/v23.1/row-level-ttl) descriptor repair. Previously, upgrading from v22.2.X to v23.1.9 incorrectly removed TTL storage parameters from tables (visible by running a `SHOW CREATE TABLE <ttl-table>;` statement) while attempting to repair table descriptors. This resulted in the node that attempted to run the TTL job crashing due to a panic caused by the missing TTL storage parameters. [#111600][#111600]
```

Comment:
This is duplicated above (and I gave suggestions on it). Maybe choose one place.